pkg/geoipupdate/config.go

```diff
@@ -33,6 +34,7 @@ func NewConfig( //nolint: gocyclo // long but breaking it up may be worse
 	defaultDatabaseDirectory,
 	databaseDirectory string,
 	verbose bool,
+	parallelism int,
```

Comment:
This package is being used as a library by third parties. If we change the public API, we should bump the module version in the `go.mod` file. That said, perhaps we could keep the same API by using [functional options](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis). Potentially we could move some of the other arguments to functional options in a future major release, e.g., `verbose`.